### PR TITLE
add test cases for emailParse.js using Jasmine

### PR DESCRIPTION
**Pull request by brandonvu12**
Edits to emailParse.js and test cases using Jasmine

### CODE REVIEW

File: project/src/main/webapp/emailParse.js [hunk 1]
```diff
@@ -11,62 +11,53 @@
...
     }
 
-    /** Parse Email Bodies to apply ML Model using Promises */
+    /** Parse Email Bodies to apply ML Model using Promises. */
     parseEmailsWithModel() {
-        // class from Gmail API js file
+        // class from Gmail API js file.
         const gmail = new gmailAPI();
-        const {question, totalObjects} = gmail;
+        const { question, totalObjects } = gmail;
```
Comment (Enoch-Chou) [inline project/src/main/webapp/emailParse.js]: I think emailObjects might make more sense than totalObjects, as totalObjects doesn't say what object it is of

File: spec/scrip-spec.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/** Testing file using Jasmine on emailParse.js. */
+let app= require("../project/src/main/webapp/emailParse.js");
+let MLClass = app.MLModelEmailParse;
+
+const modelClass = new MLClass();
+describe("getScoreToEmail",function(){
+    it("The function only returns map with ML answer and confidence score with email body for later reference",function() {
+        var currentArray = [
```
Comment (Enoch-Chou) [inline spec/scrip-spec.js]: Was it intended for you to use var, and if it was why over const and let?

File: spec/scrip-spec.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            [{text: "October 2, 2019", score: 6.968050003051758, startIndex: 85, endIndex: 100}],
+            [],
+            [{text: "October 2, 2019", score: 6.968050003051758, startIndex: 85, endIndex: 100}]
+        ];
+        let correctMap = new Map();
+        correctMap.set(6.968050003051758, { answer: 'October 2, 2019', emailBody: 'some email' })
+        var allEmailBodies = ["some email", "empty email", "some email"]
+        var value=modelClass.getScoreToEmail(currentArray, allEmailBodies);
```
Comment (Enoch-Chou) [inline spec/scrip-spec.js]: same question as above.

**Comment by brandonvu12** [comment]
PTAL. Thanks!

**Review by Enoch-Chou — APPROVED** [review]

File: spec/scrip-spec.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+/** Testing file using Jasmine on emailParse.js. */
+let app= require("../project/src/main/webapp/emailParse.js");
+let MLClass = app.MLModelEmailParse;
+
+const modelClass = new MLClass();
+describe("getScoreToEmail",function(){
+    it("The function only returns map with ML answer and confidence score with email body for later reference",function() {
```
Comment (eswarty) [inline spec/scrip-spec.js]: The description of the test will be shown right after the description of the `describe` and should read like a proper sentence. So this one would be `getScoreToEmail only returns map...` and you can remove `The function` from the test description.

**Review by eswarty — APPROVED** [review]

**Pull request merged**